### DISCUSSION TITLE
VS Code syntax highlighting breaks when the first thing in a line is an expression

### DISCUSSION THREAD

**Discussion by smartcontracts** [mdx-js/mdx#2410] [Q&A]
I'm using the [MDX plugin for VS Code](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx). I noticed that syntax highlighting is broken when the first thing in a line is an expression. For example

```
This {line} works great.
{line} two breaks everything.
```

![Screenshot from 2023-11-30 16-21-45](https://github.com/mdx-js/mdx/assets/14298799/d0de0159-63ae-499a-a53b-f9848237e3b9)

Everything after this point ends up being broken. Any help would be much appreciated!

**Comment by remcohaszing** [comment]
This is a bug in https://github.com/wooorm/markdown-tm-language.

**Reply by smartcontracts** [reply]
Ok gotcha, thank you! Should I just go ahead and open up an issue over there?

**Reply by remcohaszing** [reply]
Yes
